addons/membership/models/membership.py

```diff
@@ -0,0 +1,78 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+from odoo import api, models, fields, _
+import odoo.addons.decimal_precision as dp
+from odoo.exceptions import UserError
+
+STATE = [
+    ('none', 'Non Member'),
+    ('canceled', 'Cancelled Member'),
+    ('old', 'Old Member'),
+    ('waiting', 'Waiting Member'),
+    ('invoiced', 'Invoiced Member'),
+    ('free', 'Free Member'),
+    ('paid', 'Paid Member'),
+]
+
+
+class MembershipLine(models.Model):
+    _name = 'membership.membership_line'
+    _description = __doc__
+    _rec_name = 'partner_id'
+    _order = 'id desc'
+    partner_id = fields.Many2one('res.partner', 'Partner', ondelete='cascade', index=1, oldname='partner')
```

Comment:
Use string='Partner' convention wherever strings are provided for the fields(check all the model files).